regression/memory-analyzer/primitive_types/primitive_types.c

```diff
@@ -0,0 +1,42 @@
+//Copyright 2018 Author: Malte Mues <mail.mues@gmail.com>
+
+/// \file
+/// This is just a basic example.
+/// Pointer references are tested and ensured, that for example f and f_1 are
+/// pointing to the same int value location after running memory-analyzer.
+
+#include "primitive_types.h"
```

Comment:
The technical term is "pod" (plain old datatype) I'd say. Which just sounds less dismissive.

Reply:
Renamed.